Dockerfile

```diff
@@ -174,6 +174,14 @@ RUN pip install awscli==${AWSCLI_VERSION} && \
     ln -s /usr/bin/aws_bash_completer /etc/bash_completion.d/aws.sh && \
     ln -s /usr/bin/aws_completer /usr/local/bin/
 
+#
+# Install oath-toolkit
+# https://www.nongnu.org/oath-toolkit/
+# https://www.nongnu.org/oath-toolkit/oathtool.1.html
+#
+RUN echo "http://dl-cdn.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories \
+    && apk add --no-cache oath-toolkit-oathtool
```

Comment:
To avoid introducing a new repository that could cause other packages to get upgraded, let's do this instead:

```
RUN cp /etc/apk/repositories /etc/apk/repositories.testing && \
         echo "http://dl-cdn.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories.testing && \
         apk add --repositories-file=/etc/apk/repositories.testing --no-cache oath-toolkit-oathtool
```

Reply:
Let's do repository pinning. https://wiki.alpinelinux.org/wiki/Alpine_Linux_package_management

![image](https://user-images.githubusercontent.com/52489/44621632-2e2ea080-a85e-11e8-84cf-cc51335b3112.png)

Reply:
Move `echo "@testing http://dl-cdn.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories` up top. 

Then add `oath-toolkit-oathtool@testing` to `APK_PACKAGES`